src/shared/api/aragonBackendService/aragonBackendServiceError.ts

```diff
@@ -32,11 +36,30 @@ export class AragonBackendServiceError extends Error {
                 typeof (value as Record<string, unknown>).description === 'string';
 
             if (isIErrorResponse(parsedData)) {
+                // Domain-level error coming from backend (e.g. { code: "notFound", ... }).
                 return new AragonBackendServiceError(parsedData.code, parsedData.description, response.status);
             }
 
+            // Backend responded with 404 but without our standard error shape:
+            // treat it as "endpoint missing" instead of a domain not-found.
+            if (response.status === 404) {
+                return new AragonBackendServiceError(
+                    this.httpNotFoundCode,
```

Comment:
Does the Not Found code make sense for this then? I believe technically it's found but just does not contain the field we try to spread onto a new object... I think that this error handling is maybe a good improvement, but we should also actually handle this case with the construction of the { ...dao, plugins: [] } object which is actually correct and should be supported if a DAO has uninstalled all plugins.